src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -1,6 +1,9 @@
 (ns puppetlabs.trapperkeeper.services.metrics.metrics-core
-  (:import (com.codahale.metrics JmxReporter MetricRegistry))
+  (:import (com.codahale.metrics JmxReporter MetricRegistry)
+           com.fasterxml.jackson.core.JsonParseException)
```

Comment:
I think if you change the import statement to:

```clj
(:import (com.codahale.metrics JmxReporter MetricRegistry)
           (com.fasterxml.jackson.core JsonParseException))
```
it should compile, I've put this change in locally and the tests are compiling and running properly.